test-kit/components/portal-driver.ts

```diff
@@ -1,23 +1,21 @@
-import * as ReactDOM from 'react-dom';
 import {Portal} from '../../src';
 import {DriverBase} from 'test-drive-react';
 
 export class PortalTestDriver extends DriverBase {
     public static ComponentClass = Portal;
 
-    constructor(public readonly instance: Portal) {
-        super(() => ReactDOM.findDOMNode(instance));
+    private get portal(): Element | null {
+        if (!this.select('ID_SPAN')) return null;
+        const portalDomUniqueId =
```

Comment:
Misleading naming. This is portal unique id already wrapped as DOM selector. Either split into two, or rename it.

Reply:
right. renamed